dbschema/z.named.esdl

```diff
@@ -0,0 +1,24 @@
+module Mixin {
+  abstract type Named {
+    required name: str {
+      rewrite insert, update using (default::str_clean(.name));
+    };
+    
+    index on (default::str_sortable(.name));
+    
+    index fts::index on (
+      fts::with_options(
+        .name,
+        language := fts::Language.eng,
+      )
+    );
+  } 
+}
+ 
+module default {
+  function str_clean(string: str) -> optional str
+    using(
+      with t := str_trim(string, " \t\r\n")
```

Comment:
`t`?   🤔

Reply:
Random var name referenced on the next line. this probably goes back to the `sens` debate

Reply:
That was my point.  I'm not real big on one letter variables.  We're not exactly trying to save bytes here.  Why not make it more clear what it is?  🤪

Reply:
Yeah I guess my perspective is it is the output of the str_trim function, so that makes it obvious. And this is a generic function so we can't give it a more clear name.
I could change it to `trimmed`...does that really add clarity? 🤔

Reply:
I think it does, yes.  It's style and preference, so whatevz though.

Reply:
Roger, updated! 👍